Dockerfile

```diff
@@ -19,8 +20,8 @@ USER ladybugbot
 WORKDIR ${WORKDIR}
 
 # Expects a decompressed radiance folder in the build context
-COPY radiance/usr/local/radiance/bin ${WORKDIR}/bin
-COPY radiance/usr/local/radiance/lib ${WORKDIR}/lib
+COPY radiance/usr/local/radiance/bin ${BINPATH}
+COPY radiance/usr/local/radiance/lib ${RAYPATH}
 
 # Install honeybee-radiance
 COPY honeybee_radiance ${LIBRARYDIR}/honeybee_radiance
```

Comment:
@tymokvo, one more comment here! Does this compile the entrypoints for running honeybee-radiance commands from the command line? Don't we need to use pip install to create the executable files under Scripts folder?

I'm lazy to build the docker and test it myself but what happens when you run `honeybee-radiance --help` from the command line? If it gives you the right output then ignore my comment here.

Reply:
Yep, the cli help prints out. I believe `pip` handles that installation on line 38.

Reply:
You are right. Didn't see the existing lines on my phone. :|